polyglot/piranha/demo/find_replace/go/configurations/rules.toml

```diff
@@ -0,0 +1,27 @@
+# Copyright (c) 2022 Uber Technologies, Inc.
+#
+# <p>Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file
+# except in compliance with the License. You may obtain a copy of the License at
+# <p>http://www.apache.org/licenses/LICENSE-2.0
+#
+# <p>Unless required by applicable law or agreed to in writing, software distributed under the
+# License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing permissions and
+# limitations under the License.
+
+[[rules]]
+# note the quotes escape `\\"` for the string literal
+name = "remove_staleFlag"
```

Comment:
Please add examples for each rule. 
Look at how we have done it in the past for Java/kotlin rules